mindmaps-test/src/test/java/io/mindmaps/graql/internal/analytics/ScalingTestIT.java

```diff
@@ -78,7 +82,7 @@ public void setUp() throws InterruptedException {
     }
 
     @Test
-    public void testAll() throws InterruptedException, ExecutionException {
+    public void testAll() throws InterruptedException, ExecutionException, MindmapsValidationException {
```

Comment:
@fppt This one does not have any on purpose at the moment - it is an integration test and the file it dumps out is used to test the execution time and scaling of the algorithms